From a3eff2724ec4b40644a40158f7ac1b41321cd11b Mon Sep 17 00:00:00 2001
From: Li Zeng <li.zeng@intel.com>
Date: Sat, 27 Oct 2012 10:00:16 +0800
Subject: [PATCH 4/8] [PATCH] Add a flag
 OMX_INTEL_COLOR_FormatYUV420PackedSemiPlanar_Tiled

Change-Id: Iafea058d32013db2f68adeec6fadb5002c4809e5
Category: device enablement
Domain: Video
Origin: internal
Signed-off-by: Li Zeng <li.zeng@intel.com>
Reviewed-on: http://android.intel.com:8080/83203
Tested-by: Tong, BoX <box.tong@intel.com>
Reviewed-by: Tong, BoX <box.tong@intel.com>
Reviewed-by: cactus <cactus@intel.com>
Tested-by: cactus <cactus@intel.com>
---
 include/media/openmax/OMX_IVCommon.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/include/media/openmax/OMX_IVCommon.h b/include/media/openmax/OMX_IVCommon.h
index 940911e..a9d10ff 100644
--- a/include/media/openmax/OMX_IVCommon.h
+++ b/include/media/openmax/OMX_IVCommon.h
@@ -169,6 +169,8 @@ typedef enum OMX_COLOR_FORMATTYPE {
     OMX_QCOM_COLOR_FormatYUV420PackedSemiPlanar64x32Tile2m8ka = 0x7FA30C03,
     OMX_SEC_COLOR_FormatNV12Tiled = 0x7FC00002,
     OMX_QCOM_COLOR_FormatYUV420PackedSemiPlanar32m = 0x7FA30C04,
+    OMX_INTEL_COLOR_FormatYUV420PackedSemiPlanar =  0x7FA00E00,
+    OMX_INTEL_COLOR_FormatYUV420PackedSemiPlanar_Tiled =  0x7FA00F00,
     OMX_COLOR_FormatMax = 0x7FFFFFFF,
 } OMX_COLOR_FORMATTYPE;
 
-- 
2.7.4

